docs/using-ngrok-with/k8s.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Kubernetes
+---
+
+# Using ngrok with Kubernetes 
+------------
+
+## Introduction
+
+The [ngrok Ingress Controller for Kubernetes](https://github.com/ngrok/kubernetes-ingress-controller) is our official open-source controller for adding public and secure ingress traffic to your k8s clusters. If you’ve used ngrok in the past, you can think of the ngrok Ingress Controller as ngrok packaged an idiomatic k8s resource — available as helm chart, configurable via manifest (using the `kind: Ingress` construct), and compatible with k8s best practices. 
```

Comment:
Nits:
`to your k8s clusters`: technically you are giving access to the workloads/services in the cluster, not the cluster itself if that distinction is important

`you can think of the ngrok Ingress Controller as ngrok packaged an idiomatic k8s resource`: controllers in general are kubernetes components you can run to extend the functionality of k8s. Ingress controllers are specifically meant to manage `kind: Ingress` resources to convert the standard declarative Ingress Spec model into the controller _implementation specific_ method of providing ingress to a service.

Reply:
changed the copy. Feel free to make suggestions.